test/tracker-component.test.js

```diff
@@ -386,11 +386,11 @@ describe('paypal.Tracker', () => {
         const tracker = Tracker({ currencyCode: 'COWRIESHELLS', user: { email, name: userName } });
         tracker.setPropertyId(propertyId);
         expect(appendChildCalls).toBe(0);
-        tracker.cancelCart({ cartId: '__test__cartId' });
+        tracker.cancelCart();
```

Comment:
if we left the test as is, does anything break? (concerned about existing v1 integrations).